test/integration_selenium/test_upload_file_sources.py

```diff
@@ -0,0 +1,24 @@
+from .framework import (
+    selenium_test,
+    SeleniumIntegrationTestCase,
+)
+from ..integration.test_remote_files_posix import (
+    PosixFileSourceSetup,
+)
+
+
+class PosixFileSourceSeleniumIntegrationTestCase(PosixFileSourceSetup, SeleniumIntegrationTestCase):
```

Comment:
Amazing - thanks for the test :heart:.

Reply:
I'm afraid though I'll have to restructure the test, this is now breaking the integration tests :(.

Reply:
And the custom filters unit tests, maybe just throw the base class in lib/galaxy_test/driver?

Reply:
I will that, I was experimenting with pytest fixtures, but I got stuck where I got stuck the last time, combining them with unittest classes is not easy / how you're supposed to do it.